glean/pings.yaml

```diff
@@ -3,34 +3,7 @@ $schema: moz://mozilla.org/schemas/glean/pings/2-0-0
 
 page-view:
   description: |
-    A ping which is sent everytime a page is viewed.
```

Comment:
I'd recommend not deleting the data yet. Please keep the unmigrated data  around for a few releases before moving to barebone events, but other than that I'd leave this to your decision if you feel it's ok to drop this :) If so, please disregard my previous comments "not dropping"

Reply:
Aside from `page-view` which I'm keeping here for now like you suggest, none of the other custom pings are really used anywhere other than my own testing. So I think we're fine to change them now without any real data loss :)